tests/testsuite/kubevirtresource.go

```diff
@@ -206,3 +219,36 @@ func UpdateKubeVirtConfigValue(kvConfig v1.KubeVirtConfiguration) *v1.KubeVirt {
 
 	return kv
 }
+
+// waitForConfigToBeApplied waits until the KubeVirt configuration changes have been fully applied.
+func waitForConfigToBeApplied() {
+	virtClient := kubevirt.Client()
+
+	EventuallyWithOffset(3, func() bool {
```

Comment:
Are you sure about this offset?

Some lanes are failing right now and in my opinion it hides the real issue more than it helps.

Reply:
true, I am probably gonna have to use a different approach for this one, but let see if it's still relevant to be refactored in the scope of this PR